### ISSUE TITLE
Which case should we use for FONT grade? 

### ISSUE THREAD

**Issue by actuallyyun** [OpenBeta/sandbag#107]
Currently, the Font grade uses lowercase, and does not support uppercase:

`Font.getScore('7B')   Output: 0`

`Font.getScore('7b')  Output: [ 76, 77 ]`

I think it would be a good idea to use uppercase for the FONT grade to distinguish it from the French sports grade, but supports lowercase input as well. 

What do you think? @musoke

**Comment by musoke** [comment]
Yes, I would consider this a bug.  If anything it should be the opposite

**Comment by musoke** [comment]
Allowing uppercase would be easy.

Need to check how they're stored in https://github.com/openbeta/openbeta-graphql and double check with a lot of people before disallowing lowercase - I suspect that disallowing lowercase will break routes currently in the db.

**Comment by actuallyyun** [comment]
I see. Also later I noticed inconsistencies in allowing lower/upper cases in font scale methods. (`isType` is not case sensitive)
To begin with, should I delete this from README since this behavior is not consistent 
```
// Font grade is not case sensitive
console.log('Is 6a a Font Scale?',Font.isType('6a'))
```
Then we could begin a discussion about which case to allow for font grade.
What do you think?

**Comment by musoke** [comment]
I agree, remove that from the examples.
